packages/orchestration/test/examples/sendAnywhere.test.ts

```diff
@@ -121,7 +122,7 @@ test('send using arbitrary chain info', async t => {
       { Send },
       { destAddr: 'hot1destAddr', chainName },
     );
-    await E(userSeat).getOfferResult();
+    await vt.when(E(userSeat).getOfferResult());
```

Comment:
I forget where we landed on aliasing `heapVowE` to `E` in tests. These are tests that we expect developers to write and `vt.when(E(x).foo())` might be confusing.

Reply:
My gut says papering over this difference will be riskier. Let's see how it plays out